scrud_django/models.py

```diff
@@ -0,0 +1,40 @@
+"""
+This will support queries, such as:
+    GET /resource_type/schema?uri=http://schema.org/person
+    GET /resource_type/context?uri=http://schema.org/person
+
+"""
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ResourceType(models.Model):
+    type_uri = models.URLField()
+    schema = models.ForeignKey(
+        'self',
```

Comment:
Change to reference "Resource" instead of self

Reply:
thanks @trentoliphant for catching that!